### PR TITLE
Removes the skirt turtleneck and prisoner skirt from lockers

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by BeloneX** [comment]
i think travis broke

**Comment by Wilchenx** [comment]
How are skirts fetish content?

**Comment by BeloneX** [comment]
> 
> 
> How are skirts fetish content?

a prisoner skirt sure is

**Comment by St0rmC4st3r** [comment]
Travis broke. And prisoner skirts funny. If you think that they are not MRP - make a server config

**Comment by Wilchenx** [comment]
@BeloneX how is it fetish content compared to a normal skirt?

**Comment by BeloneX** [comment]
> 
> 
> @BeloneX how is it fetish content compared to a normal skirt?

Why would a prisoner have a SKIRT?

**Comment by St0rmC4st3r** [comment]
@BeloneX if you think that a prisoner skirt is fetish content then you are the one with a dirty brain :;)

**Comment by Wilchenx** [comment]
@BeloneX the prisoner obviously has a choice of garments presented to them, unless they're being forced to wear something i see no reason to call it a "fetish", moreso if we take on the "forced" scenario then the garment itself is not a fetish because it could be subsidized by another garment of the same variety, lest the effect be the same.

**Comment by BeloneX** [comment]
> 
> 
> @BeloneX the prisoner obviously has a choice of garments presented to them, unless they're being forced to wear something i see no reason to call it a "fetish", moreso if we take on the "forced" scenario then the garment itself is not a fetish because it could be subsidized by another garment of the same variety, lest the effect be the same.

You make a good point, i still find it silly for prisoners to have a skirt.

**Comment by Wilchenx** [comment]
As for the mid-thigh turtleneck sweater-dress it is a nod to old school femme fatale spy/gi-joe movies/comics.

**Comment by BeloneX** [comment]
yeah travis is def brok

> 
> 
> As for the mid-thigh turtleneck sweater-dress it is a nod to old school femme fatale spy/gi-joe movies/comics.

do we really need more useless references in this game?

**Comment by BeloneX** [comment]
> 
> 
> Travis broke. And prisoner skirts funny. If you think that they are not MRP - make a server config

i would make a config for this if i knew how to.

**Comment by BeloneX** [comment]
@qwertyquerty travis is kil what do i do

**Comment by qwertyquerty** [comment]
tbh bilary you're being picky at this point

**Comment by BeloneX** [comment]
> 
> 
> tbh bilary you're being picky at this point

okay

**Comment by BeloneX** [comment]
@qwertyquerty  travis is no longer fuck

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -75,16 +75,6 @@
...
-/obj/item/clothing/under/rank/prisoner/skirt
-	name = "prison jumpskirt"
-	desc = "It's standardised Nanotrasen prisoner-wear. Its suit sensors are stuck in the \"Fully On\" position."
-	icon_state = "prisoner_skirt"
-	item_state = "o_suit"
-	item_color = "prisoner_skirt"
-	body_parts_covered = CHEST|GROIN|ARMS
-	can_adjust = FALSE
-	fitted = FEMALE_UNIFORM_TOP
-
```
Comment (Wilchenx) [inline code/modules/clothing/under/miscellaneous.dm]: no need to remove the item from being spawnable by admins

File: code/modules/clothing/under/syndicate.dm [hunk 2]
```diff
@@ -8,17 +8,6 @@
...
-	name = "tactical skirtleneck"
-	desc = "A non-descript and slightly suspicious looking skirtleneck."
-	icon_state = "syndicate_skirt"
-	item_state = "bl_suit"
-	item_color = "syndicate_skirt"
-	has_sensor = NO_SENSORS
-	armor = list("melee" = 10, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 40)
-	alt_covers_chest = TRUE
-	fitted = FEMALE_UNIFORM_TOP
-
```
Comment (Wilchenx) [inline code/modules/clothing/under/syndicate.dm]: same here

**Comment by Wilchenx** [comment]
Also you're making your changes on your master fork

**Comment by BeloneX** [comment]
> 
> 
> Also you're making your changes on your master fork

because im lazy

**Comment by BeloneX** [comment]
these commits are a mess, jesus christ.

File: code/modules/clothing/under/miscellaneous.dm (hunk 1 above)
Comment (ghost) [reply code/modules/clothing/under/miscellaneous.dm]: alright, sure.

**Comment by wesoda25** [comment]
loled at wilchen

**Comment by Wilchenx** [comment]
obsessed

**Pull request closed without merging**